includes/RequestWiki/RequestWikiRequestViewer.php

```diff
@@ -113,18 +114,24 @@ public function getFormDescriptor(
 		];
 
 		foreach ( $request->getComments() as $comment ) {
-			$formDescriptor['comment' . $comment['timestamp'] ] = [
+			$formDescriptor['comment' . $comment['timestamp']] = [
 				'type' => 'textarea',
 				'readonly' => true,
 				'section' => 'comments',
 				'rows' => 4,
 				// @phan-suppress-next-line SecurityCheck-XSS
-				'label' => wfMessage( 'requestwikiqueue-request-header-wikicreatorcomment-withtimestamp' )->rawParams( $comment['user']->getName() )->params( $context->getLanguage()->timeanddate( $comment['timestamp'], true ) )->text(),
+				'label' => wfMessage( 'requestwikiqueue-request-header-wikicreatorcomment-withtimestamp' )->rawParams(
+					$comment['user']->getName()
+				)->params( $context->getLanguage()->timeanddate( $comment['timestamp'], true ) )->text(),
 				'default' => $comment['comment'],
 			];
 		}
 
-		if ( $permissionManager->userHasRight( $userR, 'createwiki' ) && !$userR->getBlock() || $userR->getId() == $request->requester->getId() ) {
+		if (
+			$permissionManager->userHasRight( $userR, 'createwiki' ) &&
+			!$userR->getBlock() ||
+			$userR->getId() == $request->requester->getId()
+		) {
```

Comment:
Seeing as this is meant for wiki creators(ie has createwiki permission) only and not requesting users who can also edit requests, and this logic isn't really reused, I'm leaving this as it.

Reply:
To clarify to prevent incorrect suggestions, this file handles all the viewing logic, public and wiki creator alike. This specific notice is set to only display for users with createwiki right, that's not a file wide thing